src/DrawioAppServer/DrawioWebviewInitializer.ts

```diff
@@ -23,8 +23,21 @@ export class DrawioWebviewInitializer {
 	): Promise<CustomDrawioInstance> {
 		const config = this.config.getConfig(uri);
 
+		const localResourceRoots = config.customPlugins.map((p) => {
+			return Uri.file(path.dirname(p));
+		});
+		const extensionPath = extensions.getExtension("hediet.vscode-drawio")
+			?.extensionPath;
+		if (extensionPath) {
```

Comment:
Why can extensionPath be falsy?

Reply:
if the extension doesn't exist under this name

Reply:
Ah, another reason why the result of extensions.getExtension("...") should be injected through construction injection!